suit_generator/cmd_encrypt.py

```diff
@@ -0,0 +1,217 @@
+#
+# Copyright (c) 2025 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+"""Generate encryption artifacts for SUIT."""
+
+import uuid
+import logging
+import importlib.util
+import sys
+import os
+from argparse import RawTextHelpFormatter
+from pathlib import Path
+from suit_generator.suit_encrypt_script_base import (
+    SuitEncryptorBase,
+    SuitDigestAlgorithms,
+    SuitKWAlgorithms,
+)
+from suit_generator.exceptions import GeneratorError
+
+ENCRYPT_AND_GENERATE_FIRMWARE_CMD = "encrypt-and-generate"
+GENERATE_INFO_FIRMWARE_CMD = "generate-info"
+
+log = logging.getLogger(__name__)
+
+ENCRYPT_CMD = "encrypt"
+
+
+def _import_module_from_path(module_name, file_path):
+    # Helper function to import a python module from a file path.
+    spec = importlib.util.spec_from_file_location(module_name, file_path)
+    module = importlib.util.module_from_spec(spec)
+    sys.modules[module_name] = module
+    spec.loader.exec_module(module)
+    return module
+
+
+def _import_encryptor(encrypt_script: Path) -> SuitEncryptorBase:
```

Comment:
nit: `_import_module_from_path` contains a comment (which could be a docstring), but there is neither a comment nor a docstring. It would be nice to have something to be consistent

Reply:
Ok, fixed